src/explore-education-statistics-frontend/src/modules/find-statistics/PublicationReleasePage.tsx

```diff
@@ -68,22 +70,22 @@ class PublicationReleasePage extends Component<Props> {
           <div className="govuk-grid-column-two-thirds">
             <div className="govuk-grid-row">
               <div className="govuk-grid-column-three-quarters">
-                {!release ? (
+                {data.latestRelease ? (
                   <strong className="govuk-tag govuk-!-margin-right-6">
                     {' '}
                     This is the latest data{' '}
                   </strong>
                 ) : (
-                  <p>
-                    This is data from {release},{' '}
-                    <Link
-                      className="dfe-print-hidden"
-                      unvisited
-                      to={`/find-statistics/${data.publication.slug}`}
-                    >
-                      view latest release
-                    </Link>
-                  </p>
+                  <Link
+                    className="dfe-print-hidden"
+                    unvisited
+                    to={`/find-statistics/${data.publication.slug}`}
+                  >
+                    View latest data:{' '}
+                    <span className="govuk-!-font-weight-bold">
+                      {data.publication.releases.slice(-1)[0].yearTitle}
```

Comment:
I'd check with marv but you might want to combine coverageTitle with the yearTitle here so that you get `Academic Year 2016/17` rather than just `2016/17`